### PR TITLE
Bump Cluster API Visualizer to v1.3.0

### PR DESCRIPTION
**Pull request by Jont828**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**: Bump Cluster API Visualizer to v1.3.0

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

- [ ] cherry-pick candidate

**TODOs**:

- [ ] squashed commits
- [ ] includes documentation
- [ ] adds unit tests

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by Jont828** [comment]
@nojnhuh @willie-yao @nawazkh Could you try this out to see if the ASO CRDs show up properly for you?

File: azure/services/aso/aso.go [hunk 1]
```diff
@@ -175,6 +176,8 @@ func (r *reconciler[T]) CreateOrUpdateResource(ctx context.Context, spec azure.A
 	if labels == nil {
 		labels = make(map[string]string)
 	}
+	labels[clusterv1.ClusterNameLabel] = r.clusterName
```
Comment (willie-yao) [inline azure/services/aso/aso.go:179]: Is this label not already set somewhere else?

**Review by willie-yao — COMMENTED** [review]
Looks good, it works for me! Just one question from my end

File: azure/services/aso/aso.go (hunk 1 above)
Comment (Jont828) [reply azure/services/aso/aso.go:179]: No, the label previously did not exist on any of the ASO resources.

**Review by willie-yao — APPROVED** [review]
/lgtm

**Review by mboersma — APPROVED** [review]
/lgtm++
/assign @nojnhuh

**Pull request merged**